src/services/meta/meta.service.ts

```diff
@@ -0,0 +1,44 @@
+import useNetwork from '@/composables/useNetwork';
+import { Config } from '@/lib/config';
+import { RouteLocationNormalized } from 'vue-router';
+import { ROUTE_META_DATA } from './meta.constants';
+
+interface IMetaService {
+  setMeta(route: RouteLocationNormalized | string): void;
+}
+
+class MetaService implements IMetaService {
+  networkConfig: Config;
+  constructor() {
+    const { networkConfig: _networkConfig } = useNetwork();
+    this.networkConfig = _networkConfig;
+  }
```

Comment:
I didn't realise it was to use in a service. For this it's probably best to do it like this since that's how we do it elsewhere. 
```suggestion
  constructor(private readonly config = configService) {}
```
Then within the class you can do:
```
this.config.network.name;
```

Reply:
ah, yeah, it's much better. thanks

Reply:
ah, yeah, it's much better. thanks